### ISSUE TITLE
Missing hints used by bootloaders

### ISSUE THREAD

**Issue by MauroToscano** [starkware-libs/cairo-lang#171]
It seems that some of the hints that the bootloaders use are missing

For example:
https://github.com/starkware-libs/cairo-lang/blob/27a157d761ae49b242026bcbe5fca6e60c1e98bd/src/starkware/cairo/bootloaders/simple_bootloader/simple_bootloader.cairo#L19-L22

`
%{
    from starkware.cairo.bootloaders.simple_bootloader.objects import SimpleBootloaderInput
    simple_bootloader_input = SimpleBootloaderInput.Schema().load(program_input)
%}
`

**Comment by Cevedale** [comment]
the error message indicates that some of the required hints used by bootloaders are missing. the hints in question are:

%builtins output pedersen range_check ecdsa bitwise ec_op keccak poseidon


you need to have the necessary declarations for these builtins in your code.  relevant modules or files importing or defining these builtins must be included or properly referenced in your script.

from starkware.cairo.common.cairo_builtins import (
    output, pedersen, range_check, ecdsa, bitwise, ec_op, keccak, poseidon
)

**Comment by Okm165** [comment]
I found out that there at least one more missing file
Namely [this](https://github.com/Okm165/cairo-lang/blob/de1b235b44b559883e54e24922ce3bf2fe7c361f/src/starkware/cairo/bootloaders/simple_bootloader/simple_bootloader.cairo#L28-L32) line needs utils.py in simple_bootloader dir, for configure_fact_topologies & write_to_fact_topologies_file func
Also needed for core functionalities like:
[load_program](https://github.com/Okm165/cairo-lang/blob/1883aed2c7b7e697de9f1880d459b5c63882bfab/src/starkware/cairo/bootloaders/simple_bootloader/execute_task.cairo#L61-L69)
[load_cairo_pie](https://github.com/Okm165/cairo-lang/blob/1883aed2c7b7e697de9f1880d459b5c63882bfab/src/starkware/cairo/bootloaders/simple_bootloader/execute_task.cairo#L120-L154),
[prepare_output_runner](https://github.com/Okm165/cairo-lang/blob/1883aed2c7b7e697de9f1880d459b5c63882bfab/src/starkware/cairo/bootloaders/simple_bootloader/execute_task.cairo#L120-L154),
[write_return_builtins](https://github.com/Okm165/cairo-lang/blob/1883aed2c7b7e697de9f1880d459b5c63882bfab/src/starkware/cairo/bootloaders/simple_bootloader/execute_task.cairo#L175C5-L186C7)
[get_task_fact_topology](https://github.com/Okm165/cairo-lang/blob/1883aed2c7b7e697de9f1880d459b5c63882bfab/src/starkware/cairo/bootloaders/simple_bootloader/execute_task.cairo#L213C5-L225C7)

tl;dr 
cairo-lang/src/starkware/cairo/bootloaders/simple_bootloader/`utils.py`
is missing also
